shell/src/main/java/org/apache/accumulo/shell/commands/FateCommand.java

```diff
@@ -108,9 +109,8 @@ public JsonElement serialize(byte[] link, Type type, JsonSerializationContext co
     }
   }
 
-  private Option secretOption;
-  private Option statusOption;
-  private Option disablePaginationOpt;
+  private Option cancel, delete, dump, fail, list, print, secretOption, statusOption,
+      disablePaginationOpt;
```

Comment:
nit. This makes diffs much easier to understand when code changes. I would strongly prefer one line per variable.

Reply:
Addressed in c27679a

Reply:
I would love to enforce it with our formatter style, but I don't think it's possible.

Reply:
For what it's worth, this is the recommendation from the [Code Conventions For the Java Programming Language](https://www.oracle.com/java/technologies/javase/codeconventions-declarations.html#2992)